### ISSUE TITLE
Duplicate issues created in taskwarrior

### ISSUE THREAD

**Issue by gms8994** [blampe/taskwarrior-reminders#2]
I use bugwarrior to pull issues from remote repositories in to my local taskwarrior data; using this tool then creates multiple duplicates of those bugwarrior issues within taskwarrior (I assume it writes the issue to reminders and then pulls it back in somehow and gets confused). I've attached a screenshot from taskwarrior showing more information:

<img width="561" alt="image" src="https://user-images.githubusercontent.com/50712/86008479-db3c4c00-b9e6-11ea-8850-ddf96b56ce9c.png">

and this screenshot from the taskwarrior-reminders-sync output:

<img width="524" alt="image" src="https://user-images.githubusercontent.com/50712/86008562-f3ac6680-b9e6-11ea-913a-59d027b3afce.png">

**Comment by blampe** [comment]
Do you know if Bugwarrior preserves or overwrites user-defined metadata when it pulls to your local?

**Comment by gms8994** [comment]
I'm not completely sure, but I'll take a look at that. While I do that, here's an `info` run for the project that is mentioned in the above screenshot. Maybe it will be helpful in some way?

<details><summary>Output</summary>

```
Name          Value
------------- ------------------------------------------------------------------
ID            36
Description   (bw)Is#1 - Switch to using AlphaVantage for stock prices
                2020-05-12 00:30:11 /issues/1
                2020-05-12 00:30:12 @gms8994 - Specifically,
              https://www.alphavantage.co/documentation/#latestprice for current
              prices and https://www.alphavantage.co/documentation/#dailyadj for
              dividend payments
Status        Pending
Project       finance
Entered       2020-05-12 00:23:41 (7w)
Last modified 2020-06-22 12:42:51 (8d)
Tags          synced_issue
Virtual tags  ANNOTATED ORPHAN PENDING READY TAGGED UDA UNBLOCKED PROJECT
              PRIORITY
UUID          2d068745-4613-4894-9840-59a780846a10
Urgency       6.868
Priority      M
[reminderID   9C71D35D-2720-487A-8031-BE10CF4A4150]

    project             1 *    1 =      1
    annotations       0.9 *    1 =    0.9
    tags              0.8 *    1 =    0.8
    age             0.134 *    2 =  0.268
    UDA priority.M      1 *  3.9 =    3.9
                                   ------
                                    6.868

Date               Modification
------------------ -------------------------------------------------------------
2020-05-12         Annotation of '/issues/1'
00:30:11           added.
2020-05-12         Annotation of '@gms8994 - Specifically,
00:30:12           https://www.alphavantage.co/documentation/#latestprice for
                   current prices and
                   https://www.alphavantage.co/documentation/#dailyadj for
                   dividend payments' added.
2020-06-07         Tags set to 'synced-issue'.
14:15:36           Gitlabdownvotes changed from '0' to '0.000000'.
                   Gitlabupvotes changed from '0' to '0.000000'.
                   Gitlabwip changed from '0' to '0.000000'.
2020-06-08         Tags changed from 'synced-issue' to
10:18:28           'synced-issue,synced_issue'.
2020-06-08         Tags changed from 'synced-issue,synced_issue' to
10:20:40           'synced_issue'.
2020-06-22         Gitlabauthor deleted.
12:42:51           Gitlabcreatedon deleted.
                   Gitlabdescription deleted.
                   Gitlabdownvotes deleted.
                   Gitlabnamespace deleted.
                   Gitlabnumber deleted.
                   Gitlabrepo deleted.
                   Gitlabstate deleted.
                   Gitlabtitle deleted.
                   Gitlabtype deleted.
                   Gitlabupdatedat deleted.
                   Gitlabupvotes deleted.
                   Gitlaburl deleted.
                   Gitlabwip deleted.
                   ReminderID set to '9C71D35D-2720-487A-8031-BE10CF4A4150'.


Name          Value
------------- ------------------------------------------------------------------
ID            58
Description   (bw)Is#3 - New server error
                2020-06-27 13:28:11 /issues/3
Status        Pending
Project       finance
Entered       2020-06-27 11:59:50 (3d)
Last modified 2020-06-27 13:28:30 (3d)
Tags          synced_issue
Virtual tags  ANNOTATED ORPHAN PENDING READY TAGGED UDA UNBLOCKED PROJECT
              PRIORITY
UUID          b39804f0-45e6-4ce9-bdb6-b1d06c017946
Urgency       6.516
Priority      M
[reminderID   0E73659A-12FB-4095-8417-1BBFC048AEAB]

    project             1 *    1 =      1
    annotations       0.8 *    1 =    0.8
    tags              0.8 *    1 =    0.8
    age             0.008 *    2 =  0.016
    UDA priority.M      1 *  3.9 =    3.9
                                   ------
                                    6.516

Date                Modification
------------------- ------------------------------------------------------------
2020-06-27 13:28:11 Annotation of '/issues/3'
                    added.
                    Gitlabdownvotes changed from '0' to '0.000000'.
                    Gitlabupvotes changed from '0' to '0.000000'.
                    Gitlabwip changed from '0' to '0.000000'.
2020-06-27 13:28:29 Gitlabauthor deleted.
                    Gitlabcreatedon deleted.
                    Gitlabdescription deleted.
                    Gitlabdownvotes deleted.
                    Gitlabnamespace deleted.
                    Gitlabnumber deleted.
                    Gitlabrepo deleted.
                    Gitlabstate deleted.
                    Gitlabtitle deleted.
                    Gitlabtype deleted.
                    Gitlabupdatedat deleted.
                    Gitlabupvotes deleted.
                    Gitlaburl deleted.
                    Gitlabwip deleted.
                    ReminderID set to 'DC8D1307-84A6-4367-9C32-D0A2748A00A3'.
2020-06-27 13:28:30 ReminderID changed from
                    'DC8D1307-84A6-4367-9C32-D0A2748A00A3' to
                    '0E73659A-12FB-4095-8417-1BBFC048AEAB'.


Name          Value
------------- ------------------------------------------------------------------
ID            59
Description   (bw)Is#1 - Switch to using AlphaVantage for stock prices
                2020-06-27 13:28:11 /issues/1
                2020-06-27 13:28:12 @gms8994 - Specifically,
              https://www.alphavantage.co/documentation/#latestprice for current
              prices and https://www.alphavantage.co/documentation/#dailyadj for
              dividend payments
Status        Pending
Project       finance
Entered       2020-05-12 00:23:41 (7w)
Last modified 2020-06-27 13:28:34 (3d)
Tags          synced_issue
Virtual tags  ANNOTATED ORPHAN PENDING READY TAGGED UDA UNBLOCKED PROJECT
              PRIORITY
UUID          e9b0c9f7-d7b9-4bd8-bd79-a98572fae13c
Urgency       6.868
Priority      M
[reminderID   A5FDE4E1-FEFC-422D-835B-F0CE5EDD71B4]

    project             1 *    1 =      1
    annotations       0.9 *    1 =    0.9
    tags              0.8 *    1 =    0.8
    age             0.134 *    2 =  0.268
    UDA priority.M      1 *  3.9 =    3.9
                                   ------
                                    6.868

Date               Modification
------------------ -------------------------------------------------------------
2020-06-27         Annotation of '/issues/1'
13:28:11           added.
                   Gitlabdownvotes changed from '0' to '0.000000'.
                   Gitlabupvotes changed from '0' to '0.000000'.
                   Gitlabwip changed from '0' to '0.000000'.
2020-06-27         Annotation of '@gms8994 - Specifically,
13:28:12           https://www.alphavantage.co/documentation/#latestprice for
                   current prices and
                   https://www.alphavantage.co/documentation/#dailyadj for
                   dividend payments' added.
2020-06-27         Gitlabauthor deleted.
13:28:32           Gitlabcreatedon deleted.
                   Gitlabdescription deleted.
                   Gitlabdownvotes deleted.
                   Gitlabnamespace deleted.
                   Gitlabnumber deleted.
                   Gitlabrepo deleted.
                   Gitlabstate deleted.
                   Gitlabtitle deleted.
                   Gitlabtype deleted.
                   Gitlabupdatedat deleted.
                   Gitlabupvotes deleted.
                   Gitlaburl deleted.
                   Gitlabwip deleted.
                   ReminderID set to '3CB8B409-35D3-4599-9A2D-B6DF597DF757'.
2020-06-27         ReminderID changed from
13:28:34           '3CB8B409-35D3-4599-9A2D-B6DF597DF757' to
                   'A5FDE4E1-FEFC-422D-835B-F0CE5EDD71B4'.


Name          Value
------------- ------------------------------------------------------------------
ID            71
Description   (bw)Is#3 - New server error
                2020-06-27 13:30:08 /issues/3
                2020-06-27 13:30:09 @gms8994 - changed the description
Status        Pending
Project       finance
Entered       2020-06-27 11:59:50 (3d)
Last modified 2020-06-27 13:30:27 (3d)
Tags          synced_issue
Virtual tags  ANNOTATED ORPHAN PENDING READY TAGGED UDA UNBLOCKED PROJECT
              PRIORITY
UUID          9bf2156f-3f29-4bd5-a8d1-0116a8bf7dff
Urgency       6.616
Priority      M
[reminderID   16D5DFD7-0D0C-451C-AA30-33D99E7FAB1D]

    project             1 *    1 =      1
    annotations       0.9 *    1 =    0.9
    tags              0.8 *    1 =    0.8
    age             0.008 *    2 =  0.016
    UDA priority.M      1 *  3.9 =    3.9
                                   ------
                                    6.616

Date                Modification
------------------- ------------------------------------------------------------
2020-06-27 13:30:08 Annotation of '/issues/3'
                    added.
                    Gitlabdownvotes changed from '0' to '0.000000'.
                    Gitlabupvotes changed from '0' to '0.000000'.
                    Gitlabwip changed from '0' to '0.000000'.
2020-06-27 13:30:08 Annotation of '@gms8994 - changed the description' added.
2020-06-27 13:30:24 Gitlabauthor deleted.
                    Gitlabcreatedon deleted.
                    Gitlabdescription deleted.
                    Gitlabdownvotes deleted.
                    Gitlabnamespace deleted.
                    Gitlabnumber deleted.
                    Gitlabrepo deleted.
                    Gitlabstate deleted.
                    Gitlabtitle deleted.
                    Gitlabtype deleted.
                    Gitlabupdatedat deleted.
                    Gitlabupvotes deleted.
                    Gitlaburl deleted.
                    Gitlabwip deleted.
                    ReminderID set to '48002915-985A-4DC9-BAAE-26FE9CEC21C9'.
2020-06-27 13:30:27 ReminderID changed from
                    '48002915-985A-4DC9-BAAE-26FE9CEC21C9' to
                    '16D5DFD7-0D0C-451C-AA30-33D99E7FAB1D'.


Name          Value
------------- ------------------------------------------------------------------
ID            72
Description   (bw)Is#1 - Switch to using AlphaVantage for stock prices
                2020-06-27 13:30:09 /issues/1
                2020-06-27 13:30:10 @gms8994 - Specifically,
              https://www.alphavantage.co/documentation/#latestprice for current
              prices and https://www.alphavantage.co/documentation/#dailyadj for
              dividend payments
Status        Pending
Project       finance
Entered       2020-05-12 00:23:41 (7w)
Last modified 2020-06-27 13:30:30 (3d)
Tags          synced_issue
Virtual tags  ANNOTATED ORPHAN PENDING READY TAGGED UDA UNBLOCKED PROJECT
              PRIORITY
UUID          01a83add-cc16-4dde-be85-928d1ba9c02f
Urgency       6.868
Priority      M
[reminderID   8A539958-B496-4213-A6BC-DE5E02B15778]

    project             1 *    1 =      1
    annotations       0.9 *    1 =    0.9
    tags              0.8 *    1 =    0.8
    age             0.134 *    2 =  0.268
    UDA priority.M      1 *  3.9 =    3.9
                                   ------
                                    6.868

Date               Modification
------------------ -------------------------------------------------------------
2020-06-27         Annotation of '/issues/1'
13:30:09           added.
                   Gitlabdownvotes changed from '0' to '0.000000'.
                   Gitlabupvotes changed from '0' to '0.000000'.
                   Gitlabwip changed from '0' to '0.000000'.
2020-06-27         Annotation of '@gms8994 - Specifically,
13:30:09           https://www.alphavantage.co/documentation/#latestprice for
                   current prices and
                   https://www.alphavantage.co/documentation/#dailyadj for
                   dividend payments' added.
2020-06-27         Gitlabauthor deleted.
13:30:27           Gitlabcreatedon deleted.
                   Gitlabdescription deleted.
                   Gitlabdownvotes deleted.
                   Gitlabnamespace deleted.
                   Gitlabnumber deleted.
                   Gitlabrepo deleted.
                   Gitlabstate deleted.
                   Gitlabtitle deleted.
                   Gitlabtype deleted.
                   Gitlabupdatedat deleted.
                   Gitlabupvotes deleted.
                   Gitlaburl deleted.
                   Gitlabwip deleted.
                   ReminderID set to '57C06AAD-942F-4948-9053-86DE3964D3F3'.
2020-06-27         ReminderID changed from
13:30:30           '57C06AAD-942F-4948-9053-86DE3964D3F3' to
                   '8A539958-B496-4213-A6BC-DE5E02B15778'.


Name          Value
------------- ------------------------------------------------------------------
ID            85
Description   (bw)Is#3 - New server error
                2020-06-27 13:45:09 /issues/3
                2020-06-27 13:45:10 @gms8994 - changed the description
Status        Pending
Project       finance
Entered       2020-06-27 11:59:50 (3d)
Last modified 2020-06-27 13:45:26 (3d)
Tags          synced_issue
Virtual tags  ANNOTATED ORPHAN PENDING READY TAGGED UDA UNBLOCKED PROJECT
              PRIORITY
UUID          d3e74ac6-6349-455b-832d-68d9042d13d3
Urgency       6.616
Priority      M
[reminderID   098924D3-DE0C-4A42-B75A-E91E555FC183]

    project             1 *    1 =      1
    annotations       0.9 *    1 =    0.9
    tags              0.8 *    1 =    0.8
    age             0.008 *    2 =  0.016
    UDA priority.M      1 *  3.9 =    3.9
                                   ------
                                    6.616

Date                Modification
------------------- ------------------------------------------------------------
2020-06-27 13:45:09 Annotation of '/issues/3'
                    added.
                    Gitlabdownvotes changed from '0' to '0.000000'.
                    Gitlabupvotes changed from '0' to '0.000000'.
                    Gitlabwip changed from '0' to '0.000000'.
2020-06-27 13:45:09 Annotation of '@gms8994 - changed the description' added.
2020-06-27 13:45:26 Gitlabauthor deleted.
                    Gitlabcreatedon deleted.
                    Gitlabdescription deleted.
                    Gitlabdownvotes deleted.
                    Gitlabnamespace deleted.
                    Gitlabnumber deleted.
                    Gitlabrepo deleted.
                    Gitlabstate deleted.
                    Gitlabtitle deleted.
                    Gitlabtype deleted.
                    Gitlabupdatedat deleted.
                    Gitlabupvotes deleted.
                    Gitlaburl deleted.
                    Gitlabwip deleted.
                    ReminderID set to 'A87977C3-0E6B-4D0A-975A-B20C2729DF94'.
2020-06-27 13:45:26 ReminderID changed from
                    'A87977C3-0E6B-4D0A-975A-B20C2729DF94' to
                    '098924D3-DE0C-4A42-B75A-E91E555FC183'.
2020-06-27 13:45:27 ReminderID changed from
                    '098924D3-DE0C-4A42-B75A-E91E555FC183' to
                    'A87977C3-0E6B-4D0A-975A-B20C2729DF94'.
2020-06-27 13:45:27 ReminderID changed from
                    'A87977C3-0E6B-4D0A-975A-B20C2729DF94' to
                    '098924D3-DE0C-4A42-B75A-E91E555FC183'.


Name          Value
------------- ------------------------------------------------------------------
ID            86
Description   (bw)Is#1 - Switch to using AlphaVantage for stock prices
                2020-06-27 13:45:09 /issues/1
                2020-06-27 13:45:10 @gms8994 - Specifically,
              https://www.alphavantage.co/documentation/#latestprice for current
              prices and https://www.alphavantage.co/documentation/#dailyadj for
              dividend payments
Status        Pending
Project       finance
Entered       2020-05-12 00:23:41 (7w)
Last modified 2020-06-27 13:45:30 (3d)
Tags          synced_issue
Virtual tags  ANNOTATED ORPHAN PENDING READY TAGGED UDA UNBLOCKED PROJECT
              PRIORITY
UUID          f9a59c14-2b84-404a-8e70-bf98040f7bcb
Urgency       6.868
Priority      M
[reminderID   734992A5-585C-46E8-984B-E92AEF2E2FC0]

    project             1 *    1 =      1
    annotations       0.9 *    1 =    0.9
    tags              0.8 *    1 =    0.8
    age             0.134 *    2 =  0.268
    UDA priority.M      1 *  3.9 =    3.9
                                   ------
                                    6.868

Date               Modification
------------------ -------------------------------------------------------------
2020-06-27         Annotation of '/issues/1'
13:45:09           added.
                   Gitlabdownvotes changed from '0' to '0.000000'.
                   Gitlabupvotes changed from '0' to '0.000000'.
                   Gitlabwip changed from '0' to '0.000000'.
2020-06-27         Annotation of '@gms8994 - Specifically,
13:45:10           https://www.alphavantage.co/documentation/#latestprice for
                   current prices and
                   https://www.alphavantage.co/documentation/#dailyadj for
                   dividend payments' added.
2020-06-27         Gitlabauthor deleted.
13:45:30           Gitlabcreatedon deleted.
                   Gitlabdescription deleted.
                   Gitlabdownvotes deleted.
                   Gitlabnamespace deleted.
                   Gitlabnumber deleted.
                   Gitlabrepo deleted.
                   Gitlabstate deleted.
                   Gitlabtitle deleted.
                   Gitlabtype deleted.
                   Gitlabupdatedat deleted.
                   Gitlabupvotes deleted.
                   Gitlaburl deleted.
                   Gitlabwip deleted.
                   ReminderID set to '5C5F7B45-1BFB-49C9-8002-666E8473CA6D'.
2020-06-27         ReminderID changed from
13:45:30           '5C5F7B45-1BFB-49C9-8002-666E8473CA6D' to
                   '734992A5-585C-46E8-984B-E92AEF2E2FC0'.
2020-06-27         ReminderID changed from
13:45:31           '734992A5-585C-46E8-984B-E92AEF2E2FC0' to
                   '5C5F7B45-1BFB-49C9-8002-666E8473CA6D'.
2020-06-27         ReminderID changed from
13:45:31           '5C5F7B45-1BFB-49C9-8002-666E8473CA6D' to
                   '734992A5-585C-46E8-984B-E92AEF2E2FC0'.


Name          Value
------------- ------------------------------------------------------------------
ID            99
Description   (bw)Is#3 - New server error
                2020-06-28 20:30:14 /issues/3
                2020-06-28 20:30:15 @gms8994 - changed the description
                2020-06-28 20:30:16 @gms8994 - changed the description
Status        Pending
Project       finance
Entered       2020-06-27 11:59:50 (3d)
Last modified 2020-06-28 20:30:28 (1d)
Tags          synced_issue
Virtual tags  ANNOTATED ORPHAN PENDING READY TAGGED UDA UNBLOCKED PROJECT
              PRIORITY
UUID          a3e25081-2f07-430d-a828-ee097abdc96f
Urgency       6.716
Priority      M
[reminderID   19E2ED79-7DE7-4547-B714-502D875F59FC]

    project             1 *    1 =      1
    annotations         1 *    1 =      1
    tags              0.8 *    1 =    0.8
    age             0.008 *    2 =  0.016
    UDA priority.M      1 *  3.9 =    3.9
                                   ------
                                    6.716

Date                Modification
------------------- ------------------------------------------------------------
2020-06-28 20:30:14 Annotation of '/issues/3'
                    added.
                    Gitlabdownvotes changed from '0' to '0.000000'.
                    Gitlabupvotes changed from '0' to '0.000000'.
                    Gitlabwip changed from '0' to '0.000000'.
2020-06-28 20:30:15 Annotation of '@gms8994 - changed the description' added.
2020-06-28 20:30:15 Annotation of '@gms8994 - changed the description' added.
2020-06-28 20:30:27 Gitlabauthor deleted.
                    Gitlabcreatedon deleted.
                    Gitlabdescription deleted.
                    Gitlabdownvotes deleted.
                    Gitlabnamespace deleted.
                    Gitlabnumber deleted.
                    Gitlabrepo deleted.
                    Gitlabstate deleted.
                    Gitlabtitle deleted.
                    Gitlabtype deleted.
                    Gitlabupdatedat deleted.
                    Gitlabupvotes deleted.
                    Gitlaburl deleted.
                    Gitlabwip deleted.
                    ReminderID set to '06E654B4-9005-4394-B660-D6AC9D73FE89'.
2020-06-28 20:30:28 ReminderID changed from
                    '06E654B4-9005-4394-B660-D6AC9D73FE89' to
                    '19E2ED79-7DE7-4547-B714-502D875F59FC'.


Name          Value
------------- ------------------------------------------------------------------
ID            100
Description   (bw)Is#1 - Switch to using AlphaVantage for stock prices
                2020-06-28 20:30:15 /issues/1
                2020-06-28 20:30:16 @gms8994 - Specifically,
              https://www.alphavantage.co/documentation/#latestprice for current
              prices and https://www.alphavantage.co/documentation/#dailyadj for
              dividend payments
Status        Pending
Project       finance
Entered       2020-05-12 00:23:41 (7w)
Last modified 2020-06-28 20:30:31 (1d)
Tags          synced_issue
Virtual tags  ANNOTATED ORPHAN PENDING READY TAGGED UDA UNBLOCKED PROJECT
              PRIORITY
UUID          4b9d0379-e133-4921-8d18-bbae77d69080
Urgency       6.868
Priority      M
[reminderID   B7BB5104-17FB-40DA-A8CE-D69B1B183B4F]

    project             1 *    1 =      1
    annotations       0.9 *    1 =    0.9
    tags              0.8 *    1 =    0.8
    age             0.134 *    2 =  0.268
    UDA priority.M      1 *  3.9 =    3.9
                                   ------
                                    6.868

Date               Modification
------------------ -------------------------------------------------------------
2020-06-28         Annotation of '/issues/1'
20:30:15           added.
                   Gitlabdownvotes changed from '0' to '0.000000'.
                   Gitlabupvotes changed from '0' to '0.000000'.
                   Gitlabwip changed from '0' to '0.000000'.
2020-06-28         Annotation of '@gms8994 - Specifically,
20:30:15           https://www.alphavantage.co/documentation/#latestprice for
                   current prices and
                   https://www.alphavantage.co/documentation/#dailyadj for
                   dividend payments' added.
2020-06-28         Annotation '@gms8994 - Specifically,
20:30:30           https://www.alphavantage.co/documentation/#latestprice for
                   current prices and
                   https://www.alphavantage.co/documentation/#dailyadj for
                   dividend payments' deleted.
                   Gitlabauthor deleted.
                   Gitlabcreatedon deleted.
                   Gitlabdescription deleted.
                   Gitlabdownvotes deleted.
                   Gitlabnamespace deleted.
                   Gitlabnumber deleted.
                   Gitlabrepo deleted.
                   Gitlabstate deleted.
                   Gitlabtitle deleted.
                   Gitlabtype deleted.
                   Gitlabupdatedat deleted.
                   Gitlabupvotes deleted.
                   Gitlaburl deleted.
                   Gitlabwip deleted.
                   ReminderID set to '491CCA71-2AA7-4DC2-AD68-EE580BF476DD'.
2020-06-28         Annotation of '@gms8994 - Specifically,
20:30:31           https://www.alphavantage.co/documentation/#latestprice for
                   current prices and
                   https://www.alphavantage.co/documentation/#dailyadj for
                   dividend payments' added.
                   ReminderID changed from
                   '491CCA71-2AA7-4DC2-AD68-EE580BF476DD' to
                   'B7BB5104-17FB-40DA-A8CE-D69B1B183B4F'.


Name          Value
------------- ------------------------------------------------------------------
ID            111
Description   (bw)Is#1 - Switch to using AlphaVantage for stock prices
                2020-06-28 20:45:06 /issues/1
                2020-06-28 20:45:07 @gms8994 - Specifically,
              https://www.alphavantage.co/documentation/#latestprice for current
              prices and https://www.alphavantage.co/documentation/#dailyadj for
              dividend payments
Status        Pending
Project       finance
Entered       2020-05-12 00:23:41 (7w)
Last modified 2020-06-28 20:45:19 (1d)
Tags          synced_issue
Virtual tags  ANNOTATED ORPHAN PENDING READY TAGGED UDA UNBLOCKED PROJECT
              PRIORITY
UUID          45668611-b879-441c-beba-a1fd6f2cc8fa
Urgency       6.868
Priority      M
[reminderID   6C0A8A7F-B4F3-47E2-A2B6-585AA6BE616E]

    project             1 *    1 =      1
    annotations       0.9 *    1 =    0.9
    tags              0.8 *    1 =    0.8
    age             0.134 *    2 =  0.268
    UDA priority.M      1 *  3.9 =    3.9
                                   ------
                                    6.868

Date               Modification
------------------ -------------------------------------------------------------
2020-06-28         Annotation of '/issues/1'
20:45:06           added.
                   Gitlabdownvotes changed from '0' to '0.000000'.
                   Gitlabupvotes changed from '0' to '0.000000'.
                   Gitlabwip changed from '0' to '0.000000'.
2020-06-28         Annotation of '@gms8994 - Specifically,
20:45:07           https://www.alphavantage.co/documentation/#latestprice for
                   current prices and
                   https://www.alphavantage.co/documentation/#dailyadj for
                   dividend payments' added.
2020-06-28         Gitlabauthor deleted.
20:45:17           Gitlabcreatedon deleted.
                   Gitlabdescription deleted.
                   Gitlabdownvotes deleted.
                   Gitlabnamespace deleted.
                   Gitlabnumber deleted.
                   Gitlabrepo deleted.
                   Gitlabstate deleted.
                   Gitlabtitle deleted.
                   Gitlabtype deleted.
                   Gitlabupdatedat deleted.
                   Gitlabupvotes deleted.
                   Gitlaburl deleted.
                   Gitlabwip deleted.
                   ReminderID set to '19A11645-897C-4F6F-A55F-279F34A4BF66'.
2020-06-28         ReminderID changed from
20:45:19           '19A11645-897C-4F6F-A55F-279F34A4BF66' to
                   '6C0A8A7F-B4F3-47E2-A2B6-585AA6BE616E'.


Name          Value
------------- ------------------------------------------------------------------
ID            119
Description   (bw)Is#1 - Switch to using AlphaVantage for stock prices
                2020-06-28 21:00:07 /issues/1
                2020-06-28 21:00:08 @gms8994 - Specifically,
              https://www.alphavantage.co/documentation/#latestprice for current
              prices and https://www.alphavantage.co/documentation/#dailyadj for
              dividend payments
Status        Pending
Project       finance
Entered       2020-05-12 00:23:41 (7w)
Last modified 2020-06-28 21:00:20 (1d)
Tags          synced_issue
Virtual tags  ANNOTATED ORPHAN PENDING READY TAGGED UDA UNBLOCKED PROJECT
              PRIORITY
UUID          e2402fbe-08a3-420d-9224-2e54978302f6
Urgency       6.868
Priority      M
[reminderID   90F39C28-98DF-4879-AA08-A7736CA7E405]

    project             1 *    1 =      1
    annotations       0.9 *    1 =    0.9
    tags              0.8 *    1 =    0.8
    age             0.134 *    2 =  0.268
    UDA priority.M      1 *  3.9 =    3.9
                                   ------
                                    6.868

Date               Modification
------------------ -------------------------------------------------------------
2020-06-28         Annotation of '/issues/1'
21:00:07           added.
                   Gitlabdownvotes changed from '0' to '0.000000'.
                   Gitlabupvotes changed from '0' to '0.000000'.
                   Gitlabwip changed from '0' to '0.000000'.
2020-06-28         Annotation of '@gms8994 - Specifically,
21:00:07           https://www.alphavantage.co/documentation/#latestprice for
                   current prices and
                   https://www.alphavantage.co/documentation/#dailyadj for
                   dividend payments' added.
2020-06-28         Gitlabauthor deleted.
21:00:18           Gitlabcreatedon deleted.
                   Gitlabdescription deleted.
                   Gitlabdownvotes deleted.
                   Gitlabnamespace deleted.
                   Gitlabnumber deleted.
                   Gitlabrepo deleted.
                   Gitlabstate deleted.
                   Gitlabtitle deleted.
                   Gitlabtype deleted.
                   Gitlabupdatedat deleted.
                   Gitlabupvotes deleted.
                   Gitlaburl deleted.
                   Gitlabwip deleted.
                   ReminderID set to 'D043FD02-621C-457F-99E9-84CFF7026C7C'.
2020-06-28         ReminderID changed from
21:00:20           'D043FD02-621C-457F-99E9-84CFF7026C7C' to
                   '90F39C28-98DF-4879-AA08-A7736CA7E405'.


Name          Value
------------- ------------------------------------------------------------------
ID            127
Description   (bw)Is#1 - Switch to using AlphaVantage for stock prices
                2020-06-28 21:15:07 /issues/1
                2020-06-28 21:15:08 @gms8994 - Specifically,
              https://www.alphavantage.co/documentation/#latestprice for current
              prices and https://www.alphavantage.co/documentation/#dailyadj for
              dividend payments
Status        Pending
Project       finance
Entered       2020-05-12 00:23:41 (7w)
Last modified 2020-06-28 21:15:19 (1d)
Tags          synced_issue
Virtual tags  ANNOTATED ORPHAN PENDING READY TAGGED UDA UNBLOCKED PROJECT
              PRIORITY
UUID          60b31777-1fba-4d41-83fe-520d92af60c0
Urgency       6.868
Priority      M
[reminderID   C00DEDC5-6CF1-4942-B0F0-AE43C9F091B8]

    project             1 *    1 =      1
    annotations       0.9 *    1 =    0.9
    tags              0.8 *    1 =    0.8
    age             0.134 *    2 =  0.268
    UDA priority.M      1 *  3.9 =    3.9
                                   ------
                                    6.868

Date               Modification
------------------ -------------------------------------------------------------
2020-06-28         Annotation of '/issues/1'
21:15:07           added.
                   Gitlabdownvotes changed from '0' to '0.000000'.
                   Gitlabupvotes changed from '0' to '0.000000'.
                   Gitlabwip changed from '0' to '0.000000'.
2020-06-28         Annotation of '@gms8994 - Specifically,
21:15:07           https://www.alphavantage.co/documentation/#latestprice for
                   current prices and
                   https://www.alphavantage.co/documentation/#dailyadj for
                   dividend payments' added.
2020-06-28         Gitlabauthor deleted.
21:15:17           Gitlabcreatedon deleted.
                   Gitlabdescription deleted.
                   Gitlabdownvotes deleted.
                   Gitlabnamespace deleted.
                   Gitlabnumber deleted.
                   Gitlabrepo deleted.
                   Gitlabstate deleted.
                   Gitlabtitle deleted.
                   Gitlabtype deleted.
                   Gitlabupdatedat deleted.
                   Gitlabupvotes deleted.
                   Gitlaburl deleted.
                   Gitlabwip deleted.
                   ReminderID set to 'A9115B70-ADC7-492D-ADE3-9FCBC23EF94F'.
2020-06-28         ReminderID changed from
21:15:19           'A9115B70-ADC7-492D-ADE3-9FCBC23EF94F' to
                   'C00DEDC5-6CF1-4942-B0F0-AE43C9F091B8'.


Name          Value
------------- ------------------------------------------------------------------
ID            135
Description   (bw)Is#1 - Switch to using AlphaVantage for stock prices
                2020-06-28 21:30:07 /issues/1
                2020-06-28 21:30:08 @gms8994 - Specifically,
              https://www.alphavantage.co/documentation/#latestprice for current
              prices and https://www.alphavantage.co/documentation/#dailyadj for
              dividend payments
Status        Pending
Project       finance
Entered       2020-05-12 00:23:41 (7w)
Last modified 2020-06-28 21:30:20 (1d)
Tags          synced_issue
Virtual tags  ANNOTATED ORPHAN PENDING READY TAGGED UDA UNBLOCKED PROJECT
              PRIORITY
UUID          3a6bad59-0fc2-43a5-8456-24e1d5dadaeb
Urgency       6.868
Priority      M
[reminderID   53743749-BFB1-4DF0-A8C5-44B2CE9FCC61]

    project             1 *    1 =      1
    annotations       0.9 *    1 =    0.9
    tags              0.8 *    1 =    0.8
    age             0.134 *    2 =  0.268
    UDA priority.M      1 *  3.9 =    3.9
                                   ------
                                    6.868

Date               Modification
------------------ -------------------------------------------------------------
2020-06-28         Annotation of '/issues/1'
21:30:07           added.
                   Gitlabdownvotes changed from '0' to '0.000000'.
                   Gitlabupvotes changed from '0' to '0.000000'.
                   Gitlabwip changed from '0' to '0.000000'.
2020-06-28         Annotation of '@gms8994 - Specifically,
21:30:07           https://www.alphavantage.co/documentation/#latestprice for
                   current prices and
                   https://www.alphavantage.co/documentation/#dailyadj for
                   dividend payments' added.
2020-06-28         Gitlabauthor deleted.
21:30:18           Gitlabcreatedon deleted.
                   Gitlabdescription deleted.
                   Gitlabdownvotes deleted.
                   Gitlabnamespace deleted.
                   Gitlabnumber deleted.
                   Gitlabrepo deleted.
                   Gitlabstate deleted.
                   Gitlabtitle deleted.
                   Gitlabtype deleted.
                   Gitlabupdatedat deleted.
                   Gitlabupvotes deleted.
                   Gitlaburl deleted.
                   Gitlabwip deleted.
                   ReminderID set to '34A6F1BD-6F8E-4F70-A962-CBB3053DDA23'.
2020-06-28         ReminderID changed from
21:30:20           '34A6F1BD-6F8E-4F70-A962-CBB3053DDA23' to
                   '53743749-BFB1-4DF0-A8C5-44B2CE9FCC61'.


Name          Value
------------- ------------------------------------------------------------------
ID            143
Description   (bw)Is#1 - Switch to using AlphaVantage for stock prices
                2020-06-28 21:45:07 /issues/1
                2020-06-28 21:45:08 @gms8994 - Specifically,
              https://www.alphavantage.co/documentation/#latestprice for current
              prices and https://www.alphavantage.co/documentation/#dailyadj for
              dividend payments
Status        Pending
Project       finance
Entered       2020-05-12 00:23:41 (7w)
Last modified 2020-06-28 21:45:20 (1d)
Tags          synced_issue
Virtual tags  ANNOTATED ORPHAN PENDING READY TAGGED UDA UNBLOCKED PROJECT
              PRIORITY
UUID          dd91c8e1-768a-43de-bab4-82c5911ead04
Urgency       6.868
Priority      M
[reminderID   18437F70-0075-46FB-970E-BD19C411FE15]

    project             1 *    1 =      1
    annotations       0.9 *    1 =    0.9
    tags              0.8 *    1 =    0.8
    age             0.134 *    2 =  0.268
    UDA priority.M      1 *  3.9 =    3.9
                                   ------
                                    6.868

Date               Modification
------------------ -------------------------------------------------------------
2020-06-28         Annotation of '/issues/1'
21:45:07           added.
                   Gitlabdownvotes changed from '0' to '0.000000'.
                   Gitlabupvotes changed from '0' to '0.000000'.
                   Gitlabwip changed from '0' to '0.000000'.
2020-06-28         Annotation of '@gms8994 - Specifically,
21:45:07           https://www.alphavantage.co/documentation/#latestprice for
                   current prices and
                   https://www.alphavantage.co/documentation/#dailyadj for
                   dividend payments' added.
2020-06-28         Gitlabauthor deleted.
21:45:18           Gitlabcreatedon deleted.
                   Gitlabdescription deleted.
                   Gitlabdownvotes deleted.
                   Gitlabnamespace deleted.
                   Gitlabnumber deleted.
                   Gitlabrepo deleted.
                   Gitlabstate deleted.
                   Gitlabtitle deleted.
                   Gitlabtype deleted.
                   Gitlabupdatedat deleted.
                   Gitlabupvotes deleted.
                   Gitlaburl deleted.
                   Gitlabwip deleted.
                   ReminderID set to 'D8F38DB2-C3A1-42B2-A68F-D746242B9964'.
2020-06-28         ReminderID changed from
21:45:20           'D8F38DB2-C3A1-42B2-A68F-D746242B9964' to
                   '18437F70-0075-46FB-970E-BD19C411FE15'.


Name          Value
------------- ------------------------------------------------------------------
ID            151
Description   (bw)Is#1 - Switch to using AlphaVantage for stock prices
                2020-06-28 22:00:07 /issues/1
                2020-06-28 22:00:08 @gms8994 - Specifically,
              https://www.alphavantage.co/documentation/#latestprice for current
              prices and https://www.alphavantage.co/documentation/#dailyadj for
              dividend payments
Status        Pending
Project       finance
Entered       2020-05-12 00:23:41 (7w)
Last modified 2020-06-28 22:00:21 (1d)
Tags          synced_issue
Virtual tags  ANNOTATED ORPHAN PENDING READY TAGGED UDA UNBLOCKED PROJECT
              PRIORITY
UUID          b3fcddcf-149f-4a96-85b9-66d7456b7a53
Urgency       6.868
Priority      M
[reminderID   7F27D627-BE99-4F86-B5E3-BF63F61FA415]

    project             1 *    1 =      1
    annotations       0.9 *    1 =    0.9
    tags              0.8 *    1 =    0.8
    age             0.134 *    2 =  0.268
    UDA priority.M      1 *  3.9 =    3.9
                                   ------
                                    6.868

Date               Modification
------------------ -------------------------------------------------------------
2020-06-28         Annotation of '/issues/1'
22:00:07           added.
                   Gitlabdownvotes changed from '0' to '0.000000'.
                   Gitlabupvotes changed from '0' to '0.000000'.
                   Gitlabwip changed from '0' to '0.000000'.
2020-06-28         Annotation of '@gms8994 - Specifically,
22:00:08           https://www.alphavantage.co/documentation/#latestprice for
                   current prices and
                   https://www.alphavantage.co/documentation/#dailyadj for
                   dividend payments' added.
2020-06-28         Gitlabauthor deleted.
22:00:19           Gitlabcreatedon deleted.
                   Gitlabdescription deleted.
                   Gitlabdownvotes deleted.
                   Gitlabnamespace deleted.
                   Gitlabnumber deleted.
                   Gitlabrepo deleted.
                   Gitlabstate deleted.
                   Gitlabtitle deleted.
                   Gitlabtype deleted.
                   Gitlabupdatedat deleted.
                   Gitlabupvotes deleted.
                   Gitlaburl deleted.
                   Gitlabwip deleted.
                   ReminderID set to '020CF31F-756E-45FD-9C74-E03BE2619B45'.
2020-06-28         ReminderID changed from
22:00:21           '020CF31F-756E-45FD-9C74-E03BE2619B45' to
                   '7F27D627-BE99-4F86-B5E3-BF63F61FA415'.


Name          Value
------------- ------------------------------------------------------------------
ID            159
Description   (bw)Is#1 - Switch to using AlphaVantage for stock prices
                2020-06-28 22:17:06 /issues/1
                2020-06-28 22:17:07 @gms8994 - Specifically,
              https://www.alphavantage.co/documentation/#latestprice for current
              prices and https://www.alphavantage.co/documentation/#dailyadj for
              dividend payments
Status        Pending
Project       finance
Entered       2020-05-12 00:23:41 (7w)
Last modified 2020-06-28 22:17:18 (1d)
Tags          synced_issue
Virtual tags  ANNOTATED ORPHAN PENDING READY TAGGED UDA UNBLOCKED PROJECT
              PRIORITY
UUID          a77e57f2-bbee-4b7b-8108-e2e84a23da20
Urgency       6.868
Priority      M
[reminderID   29325565-B84A-4697-9C57-2225143714FC]

    project             1 *    1 =      1
    annotations       0.9 *    1 =    0.9
    tags              0.8 *    1 =    0.8
    age             0.134 *    2 =  0.268
    UDA priority.M      1 *  3.9 =    3.9
                                   ------
                                    6.868

Date               Modification
------------------ -------------------------------------------------------------
2020-06-28         Annotation of '/issues/1'
22:17:06           added.
                   Gitlabdownvotes changed from '0' to '0.000000'.
                   Gitlabupvotes changed from '0' to '0.000000'.
                   Gitlabwip changed from '0' to '0.000000'.
2020-06-28         Annotation of '@gms8994 - Specifically,
22:17:06           https://www.alphavantage.co/documentation/#latestprice for
                   current prices and
                   https://www.alphavantage.co/documentation/#dailyadj for
                   dividend payments' added.
2020-06-28         Gitlabauthor deleted.
22:17:16           Gitlabcreatedon deleted.
                   Gitlabdescription deleted.
                   Gitlabdownvotes deleted.
                   Gitlabnamespace deleted.
                   Gitlabnumber deleted.
                   Gitlabrepo deleted.
                   Gitlabstate deleted.
                   Gitlabtitle deleted.
                   Gitlabtype deleted.
                   Gitlabupdatedat deleted.
                   Gitlabupvotes deleted.
                   Gitlaburl deleted.
                   Gitlabwip deleted.
                   ReminderID set to 'E29017E7-8EB4-4819-8485-2F85A18D8CFE'.
2020-06-28         ReminderID changed from
22:17:18           'E29017E7-8EB4-4819-8485-2F85A18D8CFE' to
                   '29325565-B84A-4697-9C57-2225143714FC'.


Name          Value
------------- ------------------------------------------------------------------
ID            167
Description   (bw)Is#1 - Switch to using AlphaVantage for stock prices
                2020-06-28 22:30:06 /issues/1
                2020-06-28 22:30:07 @gms8994 - Specifically,
              https://www.alphavantage.co/documentation/#latestprice for current
              prices and https://www.alphavantage.co/documentation/#dailyadj for
              dividend payments
Status        Pending
Project       finance
Entered       2020-05-12 00:23:41 (7w)
Last modified 2020-06-28 22:30:20 (1d)
Tags          synced_issue
Virtual tags  ANNOTATED ORPHAN PENDING READY TAGGED UDA UNBLOCKED PROJECT
              PRIORITY
UUID          38144486-e1f0-4f9f-892b-bb4c5e4734ad
Urgency       6.868
Priority      M
[reminderID   A14BD0B3-C208-4F30-93B9-8F7F688D273F]

    project             1 *    1 =      1
    annotations       0.9 *    1 =    0.9
    tags              0.8 *    1 =    0.8
    age             0.134 *    2 =  0.268
    UDA priority.M      1 *  3.9 =    3.9
                                   ------
                                    6.868

Date               Modification
------------------ -------------------------------------------------------------
2020-06-28         Annotation of '/issues/1'
22:30:06           added.
                   Gitlabdownvotes changed from '0' to '0.000000'.
                   Gitlabupvotes changed from '0' to '0.000000'.
                   Gitlabwip changed from '0' to '0.000000'.
2020-06-28         Annotation of '@gms8994 - Specifically,
22:30:06           https://www.alphavantage.co/documentation/#latestprice for
                   current prices and
                   https://www.alphavantage.co/documentation/#dailyadj for
                   dividend payments' added.
2020-06-28         Gitlabauthor deleted.
22:30:17           Gitlabcreatedon deleted.
                   Gitlabdescription deleted.
                   Gitlabdownvotes deleted.
                   Gitlabnamespace deleted.
                   Gitlabnumber deleted.
                   Gitlabrepo deleted.
                   Gitlabstate deleted.
                   Gitlabtitle deleted.
                   Gitlabtype deleted.
                   Gitlabupdatedat deleted.
                   Gitlabupvotes deleted.
                   Gitlaburl deleted.
                   Gitlabwip deleted.
                   ReminderID set to 'CFD10C49-929C-44C0-9BC1-6808699B0256'.
2020-06-28         ReminderID changed from
22:30:20           'CFD10C49-929C-44C0-9BC1-6808699B0256' to
                   'A14BD0B3-C208-4F30-93B9-8F7F688D273F'.


Name          Value
------------- ------------------------------------------------------------------
ID            175
Description   (bw)Is#1 - Switch to using AlphaVantage for stock prices
                2020-06-28 22:45:07 /issues/1
                2020-06-28 22:45:08 @gms8994 - Specifically,
              https://www.alphavantage.co/documentation/#latestprice for current
              prices and https://www.alphavantage.co/documentation/#dailyadj for
              dividend payments
Status        Pending
Project       finance
Entered       2020-05-12 00:23:41 (7w)
Last modified 2020-06-28 22:45:20 (1d)
Tags          synced_issue
Virtual tags  ANNOTATED ORPHAN PENDING READY TAGGED UDA UNBLOCKED PROJECT
              PRIORITY
UUID          17fbcaf8-f588-40f7-85cd-f2d2ea33eaf7
Urgency       6.868
Priority      M
[reminderID   68CBA80E-AB7F-4CED-A282-033E1B734134]

    project             1 *    1 =      1
    annotations       0.9 *    1 =    0.9
    tags              0.8 *    1 =    0.8
    age             0.134 *    2 =  0.268
    UDA priority.M      1 *  3.9 =    3.9
                                   ------
                                    6.868

Date               Modification
------------------ -------------------------------------------------------------
2020-06-28         Annotation of '/issues/1'
22:45:07           added.
                   Gitlabdownvotes changed from '0' to '0.000000'.
                   Gitlabupvotes changed from '0' to '0.000000'.
                   Gitlabwip changed from '0' to '0.000000'.
2020-06-28         Annotation of '@gms8994 - Specifically,
22:45:07           https://www.alphavantage.co/documentation/#latestprice for
                   current prices and
                   https://www.alphavantage.co/documentation/#dailyadj for
                   dividend payments' added.
2020-06-28         Gitlabauthor deleted.
22:45:18           Gitlabcreatedon deleted.
                   Gitlabdescription deleted.
                   Gitlabdownvotes deleted.
                   Gitlabnamespace deleted.
                   Gitlabnumber deleted.
                   Gitlabrepo deleted.
                   Gitlabstate deleted.
                   Gitlabtitle deleted.
                   Gitlabtype deleted.
                   Gitlabupdatedat deleted.
                   Gitlabupvotes deleted.
                   Gitlaburl deleted.
                   Gitlabwip deleted.
                   ReminderID set to '2723FF1A-0879-4FC8-AD14-269719EE7A8D'.
2020-06-28         ReminderID changed from
22:45:20           '2723FF1A-0879-4FC8-AD14-269719EE7A8D' to
                   '68CBA80E-AB7F-4CED-A282-033E1B734134'.


Name          Value
------------- ------------------------------------------------------------------
ID            183
Description   (bw)Is#1 - Switch to using AlphaVantage for stock prices
                2020-06-28 23:00:14 /issues/1
                2020-06-28 23:00:15 @gms8994 - Specifically,
              https://www.alphavantage.co/documentation/#latestprice for current
              prices and https://www.alphavantage.co/documentation/#dailyadj for
              dividend payments
Status        Pending
Project       finance
Entered       2020-05-12 00:23:41 (7w)
Last modified 2020-06-28 23:00:26 (1d)
Tags          synced_issue
Virtual tags  ANNOTATED ORPHAN PENDING READY TAGGED UDA UNBLOCKED PROJECT
              PRIORITY
UUID          a6e735ce-54eb-41b6-8709-085d134b015f
Urgency       6.868
Priority      M
[reminderID   C552DC01-B773-4A89-AD5C-EB82E2FA24E5]

    project             1 *    1 =      1
    annotations       0.9 *    1 =    0.9
    tags              0.8 *    1 =    0.8
    age             0.134 *    2 =  0.268
    UDA priority.M      1 *  3.9 =    3.9
                                   ------
                                    6.868

Date               Modification
------------------ -------------------------------------------------------------
2020-06-28         Annotation of '/issues/1'
23:00:14           added.
                   Gitlabdownvotes changed from '0' to '0.000000'.
                   Gitlabupvotes changed from '0' to '0.000000'.
                   Gitlabwip changed from '0' to '0.000000'.
2020-06-28         Annotation of '@gms8994 - Specifically,
23:00:14           https://www.alphavantage.co/documentation/#latestprice for
                   current prices and
                   https://www.alphavantage.co/documentation/#dailyadj for
                   dividend payments' added.
2020-06-28         Gitlabauthor deleted.
23:00:24           Gitlabcreatedon deleted.
                   Gitlabdescription deleted.
                   Gitlabdownvotes deleted.
                   Gitlabnamespace deleted.
                   Gitlabnumber deleted.
                   Gitlabrepo deleted.
                   Gitlabstate deleted.
                   Gitlabtitle deleted.
                   Gitlabtype deleted.
                   Gitlabupdatedat deleted.
                   Gitlabupvotes deleted.
                   Gitlaburl deleted.
                   Gitlabwip deleted.
                   ReminderID set to '2153CE17-5BA9-4A56-B79F-F2390AB02EFB'.
2020-06-28         ReminderID changed from
23:00:26           '2153CE17-5BA9-4A56-B79F-F2390AB02EFB' to
                   'C552DC01-B773-4A89-AD5C-EB82E2FA24E5'.


Name                  Value
--------------------- ----------------------------------------------------------
ID                    191
Description           (bw)Is#1 - Switch to using AlphaVantage for stock prices
                        2020-06-29 09:00:08
                      /issues/1
                        2020-06-29 09:00:09 @gms8994 - Specifically,
                      https://www.alphavantage.co/documentation/#latestprice for
                      current prices and
                      https://www.alphavantage.co/documentation/#dailyadj for
                      dividend payments
Status                Pending
Project               finance
Entered               2020-05-12 00:23:41 (7w)
Last modified         2020-06-29 09:00:08 (1d)
Tags                  synced_issue
Virtual tags          ANNOTATED PENDING READY TAGGED UDA UNBLOCKED LATEST
                      PROJECT PRIORITY
UUID                  dda57adf-8c7d-4878-8d50-c17c96340954
Urgency               6.868
Gitlab Author         gms8994
Gitlab Created        1589257421
Gitlab Description    See https://www.alphavantage.co/documentation/ for more
                      information
Gitlab Downvotes      0.000000
Gitlab Namespace      gms8994
Gitlab Issue/MR       1
Gitlab Repo Slug      finance
Gitlab Issue/MR State opened
Gitlab Title          Switch to using AlphaVantage for stock prices
Gitlab Type           issue
Gitlab Updated        1589257674
Gitlab Upvotes        0.000000
Gitlab URL            /issues/1
Gitlab MR             0.000000
Work-In-Progress Flag
Priority              M

    project             1 *    1 =      1
    annotations       0.9 *    1 =    0.9
    tags              0.8 *    1 =    0.8
    age             0.134 *    2 =  0.268
    UDA priority.M      1 *  3.9 =    3.9
                                   ------
                                    6.868

Date               Modification
------------------ -------------------------------------------------------------
2020-06-29         Annotation of '/issues/1'
09:00:08           added.
                   Gitlabdownvotes changed from '0' to '0.000000'.
                   Gitlabupvotes changed from '0' to '0.000000'.
                   Gitlabwip changed from '0' to '0.000000'.
2020-06-29         Annotation of '@gms8994 - Specifically,
09:00:08           https://www.alphavantage.co/documentation/#latestprice for
                   current prices and
                   https://www.alphavantage.co/documentation/#dailyadj for
                   dividend payments' added.
```
</details>

**Comment by gms8994** [comment]
After some more debugging, it looks like taskwarrior-reminders is stripping the UDAs from the tasks that bugwarrior creates:

```
t 46 info

Name        Value                                                                                                                                                                                                                                            
ID          46                                                                                                                                                                                                                                               
Description (bw)PR#8 - Bump tokio from 0.2.20 to 0.2.22                                                                                                                                                                                                      
              2020-07-27 13:33:08 <secret>                                                                                                                                                                             
Status      Pending                                                                                                                                                                                                                                          
Project     boincview
Entered     2020-07-22 01:27:42 (5d)                                                                                                                                                                                                                         
Last        2020-07-27 13:33:08 (32s)
modified
Tags        synced_issue                                                                                                                                                                                                                                     
Virtual     ANNOTATED PENDING READY TAGGED UDA UNBLOCKED PROJECT PRIORITY
tags
UUID        21a05bf3-069d-4cc8-8e04-3151190d4b8e                                                                                                                                                                                                             
Urgency     6.527
Github Body Bumps [tokio](https://github.com/tokio-rs/tokio) from 0.2.20 to 0.2.22.                                                                                                                                                                          
            <details>
... (snipped for brevity) ...
Github      1595395662
Created
Github      gms8994                                                                                                                                                                                                                                          
Namespace                                                                                                                                                                                                                                                    
Github      8.000000
Issue/PR

```

at this point, I launch task-reminders-sync (`launchctl load -w ~/Library/LaunchAgents/com.blampe.task-reminders-sync.plist`)

```
t  21a05bf3-069d-4cc8-8e04-3151190d4b8e info

Name          Value                                                                
ID            46                                                                   
Description   (bw)PR#8 - Bump tokio from 0.2.20 to 0.2.22                          
                2020-07-27 13:33:08 <secret>   
Status        Pending                                                              
Project       boincview
Entered       2020-07-22 01:27:42 (5d)                                             
Last modified 2020-07-27 13:34:16 (1min)
Tags          synced_issue                                                         
Virtual tags  ANNOTATED ORPHAN PENDING READY TAGGED UDA UNBLOCKED PROJECT PRIORITY
UUID          21a05bf3-069d-4cc8-8e04-3151190d4b8e                                 
Urgency       6.527
Priority      M                                                                    
[reminderID   6E0ED84B-9767-48EE-925A-8CA231BB5BCE]

    project             1 *    1 =      1
    annotations       0.8 *    1 =    0.8
    tags              0.8 *    1 =    0.8
    age             0.014 *    2 =  0.027
    UDA priority.M      1 *  3.9 =    3.9
                                   ------
                                    6.527

Date                Modification                                                      
2020-07-27 13:33:08 Annotation of '<secret>' added.
                    Githubnumber changed from '8' to '8.000000'.                      
2020-07-27 13:34:16 Githubbody deleted.
                    Githubcreatedon deleted.
                    Githubnamespace deleted.
                    Githubnumber deleted.
                    Githubrepo deleted.
                    Githubstate deleted.
                    Githubtitle deleted.
                    Githubtype deleted.
                    Githubupdatedat deleted.
                    Githuburl deleted.
                    Githubuser deleted.
                    ReminderID set to '6E0ED84B-9767-48EE-925A-8CA231BB5BCE'.
```

**Comment by blampe** [comment]
> After some more debugging, it looks like taskwarrior-reminders is stripping the UDAs from the tasks that bugwarrior creates:

Ah this could make sense, great sleuthing!

I'm currently off the grid and will be for most of August, but I will try to pick this up when I'm back!

**Comment by blampe** [comment]
@gms8994 thanks for your patience, I got back and had a chance to look at this again.

I pushed what I think should resolve your issue to master. Could you try updating and let me know if you still see the problem?

**Comment by gms8994** [comment]
@blampe Apologies for the delays; it looks like that worked! However... it seems that annotations might now be being stripped when the reminderID is set (which is what's normally in the Notes field), though because I have bugwarrior running on cron, some pieces get added back (though, as you can see seconds later, deleted again 😅):

```
2020-10-05 12:10:52 Annotation of '<secret1>' added.                                                                                          
                    Gitlabdownvotes changed from '0' to '0.000000'.                                                                                                             
                    Gitlabupvotes changed from '0' to '0.000000'.                                                                                                               
                    Gitlabwip changed from '0' to '0.000000'.                                                                                                                   
2020-10-05 12:10:52 Annotation of '<secret2>' added.
2020-10-05 12:10:53 Annotation '<secret1>' deleted.                                                                                           
                    Annotation '<secret2>' deleted.                                                                         
                    ReminderID set to '74C1DC4B-FBD9-438B-A0B7-5FE72270FFCC'.                                                                                                   
2020-10-05 12:15:05 Annotation of <secret2>' added.
2020-10-05 12:15:05 Annotation of '<secret1>' added.                                                                                          
2020-10-05 12:15:07 Annotation '<secret1>' deleted.
```

I truly appreciate this tool, as it's become quite useful to me as an iPhone user!

**Comment by blampe** [comment]
@gms8994 I'm glad you're finding it useful! Your Bugwarrior use case takes it to another level I hadn't even considered 😄

Just want to clarify the behavior you're seeing with the annotations - is it something roughly like this?
* Bugwarrior syncs something from Gitlab to TW with an annotation.
* taskwarrior-reminders syncs TW to Reminders, but the reminder doesn't include any notes, right?
* taskwarrior-reminders incorrectly syncs the blank notes back from Reminders to TW
* Bugwarrior & taskwarrior-reminders then fight with each other to add/delete the annotation.

**Comment by gms8994** [comment]
@blampe finally got a chance to take a look at this again.

* Correct
* Partially correct; it looks like (hard to tell) taskwarrior-reminders is correctly setting the notes on the entry in Reminders once it pulls
* Partially correct; I'm not sure what the difference is, but taskwarrior-reminders seems to decide that there's some inconsistency with what Reminders shows vs what Taskwarrior shows as far as the annotations, and decides to update them

I tested this by pulling in this repositories issues to Taskwarrior, and here's what I see in the output for taskwarrior:

```
Date                Modification                                                                                                                         
2020-10-19 13:27:14 Annotation of 'https://github.com/blampe/taskwarrior-reminders/issues/2' added.                                                      
                    Githubnumber changed from '2' to '2.000000'.                                                                                         
2020-10-19 13:27:14 Annotation of '@blampe - Do you know if Bugwarrior preserves or overwrites user-defined metadata when it pulls to your local?' added.
2020-10-19 13:27:14 Annotation of '@gms8994 - I'm not completely sure, but I'll take a look at that. While I do that, here's an `info` run for the       
                    project that is mentioned in the above screenshot. Maybe it will be helpful in some way?<details><s...' added.                       
2020-10-19 13:27:14 Annotation of '@gms8994 - After some more debugging, it looks like taskwarrior-reminders is stripping the UDAs from the tasks that
                    bugwarrior creates:```t 46 infoName        Value                                               ...' added.
2020-10-19 13:27:15 Annotation of '@blampe - > After some more debugging, it looks like taskwarrior-reminders is stripping the UDAs from the tasks that  
                    bugwarrior creates:Ah this could make sense, great sleuthing!I'm currently off the grid and w...' added.                             
2020-10-19 13:27:15 Annotation of '@blampe - @gms8994 thanks for your patience, I got back and had a chance to look at this again.I pushed what I think
                    should resolve your issue to master. Could you try updating and let me know if you still see ...' added.
2020-10-19 13:27:15 Annotation of '@gms8994 - @blampe Apologies for the delays; it looks like that worked! However... it seems that annotations might now
                    be being stripped when the reminderID is set (which is what's normally in the Notes field), ...' added.                              
2020-10-19 13:27:15 Annotation of '@blampe - @gms8994 I'm glad you're finding it useful! Your Bugwarrior use case takes it to another level I hadn't even
                    considered ??????Just want to clarify the behavior you're seeing with the annotations - is it so...' added.
2020-10-19 13:38:32 ReminderID set to 'C3AC1AB5-A3FE-41BB-8C4D-04B27776D497'.                                                                            
                    Annotation changed to '@blampe - @gms8994 I'm glad you're finding it useful! Your Bugwarrior use case takes it to another level I    
                    hadn't even considered ������Just want to clarify the behavior you're seeing with the annotations - is it so...'.                    
2020-10-19 13:38:50 Annotation changed to '@gms8994 - I’m not completely sure, but I’ll take a look at that. While I do that, here’s an `info` run for
                    the project that is mentioned in the above screenshot. Maybe it will be helpful in some way?<details><s…'.
                    Annotation changed to '@gms8994 - After some more debugging, it looks like taskwarrior-reminders is stripping the UDAs from the tasks
                    that bugwarrior creates:```t 46 infoName        Value                                               …'.
                    Annotation changed to '@blampe - > After some more debugging, it looks like taskwarrior-reminders is stripping the UDAs from the
                    tasks that bugwarrior creates:Ah this could make sense, great sleuthing!I’m currently off the grid and w…'.
                    Annotation changed to '@blampe - @gms8994 thanks for your patience, I got back and had a chance to look at this again.I pushed what I
                    think should resolve your issue to master. Could you try updating and let me know if you still see …'.
                    Annotation changed to '@gms8994 - @blampe Apologies for the delays; it looks like that worked! However… it seems that annotations
                    might now be being stripped when the reminderID is set (which is what’s normally in the Notes field), …'.
                    Annotation changed to '@blampe - @gms8994 I’m glad you’re finding it useful! Your Bugwarrior use case takes it to another level I
                    hadn’t even considered ������Just want to clarify the behavior you’re seeing with the annotations - is it so…'.
```

Note that `2020-10-19 13:27:*` was the initial pull by bugwarrior, and then `2020-10-19 13:38:*` is when I started taskwarrior-reminders; it sets the ReminderID, and modifies some annotations.

If you have Python installed locally (which being on a Mac, you likely do), you can install and set up bugwarrior pretty easily if you wanted to test/confirm yourself - this is the config that I used to pull the issues:

```
[general]
targets = personal-github
inline_links = False
annotation_links = True
annotation_comments = True
legacy_matching = False
log.level = INFO
annotation_length = 200
description_length = 200

[personal-github]
service = github
github.username = blampe
github.include_repos = taskwarrior-reminders
# github.include_user_repos = False
github.login = <your-username>
github.token = <github-token>
github.add_tags = tw_reminder_synced
```
